activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -93,6 +98,11 @@ def travel(duration, &block)
       #   end
       #   Time.current # => Sat, 09 Nov 2013 15:34:49 EST -05:00
       def travel_to(date_or_time)
+        if block_given? && simple_stubs.stubbed?(Time, :now)
+          raise "Calling `travel_to` with a block, when we have previously already made a call to `travel_to`, can lead to confusing time stubbing. " \
+                "Please make subsequent calls to `travel_to` without a block."
```

Comment:
This text doesn't match the example shown in the changelog. There's no blocks in use there.